src/quest/manager.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, namespace: str, storage: BlobStorage, create_history: History
         self._workflow_data = []
         self._workflows: dict[str, Historian] = {}
         self._workflow_tasks: dict[str, asyncio.Task] = {}
-        self._master_serializer = MasterSerializer()
+        self._serializer: StepSerializer = MasterSerializer()
```

Comment:
The serializer dependency should be provided in the constructor - we shouldn't hard-code `MasterSerializer` here.